debian/ubuntu-advantage-tools.postinst

```diff
@@ -494,6 +512,7 @@ case "$1" in
       fi
       migrate_ubuntu_pro_beta_banner $PREVIOUS_PKG_VER
       cleanup_candidate_version_stamp_permissions
+      migrate_old_notices
```

Comment:
@CalvoM Per the discussion the other day and Lucas' comment above, lets just guard this with a version check:
```suggestion
      if dpkg --compare-versions "$PREVIOUS_PKG_VER" lt "27.13~"; then
          migrate_old_notices
      fi
```

Reply:
@CalvoM this is just a nit, so feel free to disagree, but I think we can add indentation here on the if block